tests/to-json-tests.js

```diff
@@ -927,4 +926,22 @@ test('can handle guru code block', t => {
         actions: [],
         version: expectedVersion
     });
+});
+
+test('handle empty iframe fallback message', t => {
+    var result = AdaptiveHtml.toJSON(`<iframe></iframe>`);
+    t.deepEqual(result, {
+        type: "AdaptiveCard",
+        body: [{
+            type: "Container",
+            style: "emphasis",
+            items: [{
+                type: "TextBlock",
+                text: "To view this content, please open this video in the Guru app",
+                wrap: true
+            }]
+        }],
+        actions: [],
+        version: expectedVersion
+    });
```

Comment:
is there a test for divs as well?

Reply:
yeah there are a div tests already, let me rename them for clarity